test/functional/apps/visualize/_area_chart.js

```diff
@@ -62,6 +62,31 @@ bdd.describe('visualize app', function describeIndexTests() {
   bdd.describe('area charts', function indexPatternCreation() {
     var vizName1 = 'Visualization AreaChart';
 
+    bdd.it('should save and load with special characters', function pageHeader() {
+      let vizName2 = vizName1 + '/?&=%';
+      return PageObjects.visualize.saveVisualization(vizName2)
+        .then(function (message) {
```

Comment:
At the moment, our style guide recommends putting chained methods at the same indentation level as the first line in the chain https://github.com/elastic/kibana/blob/master/style_guides/js_style_guide.md#chaining-operations.